#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'pcl_filter_utils'
try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *;
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *;
from math import pi
gen = ParameterGenerator ()

gen.add("min_distance", double_t, 0,"min distance", 0.6, 0.0, 10.0)
gen.add("max_distance", double_t, 0,"max distance", 2.0, 0.0, 10.0)

exit (gen.generate (PACKAGE, "pcl_filter_utils", "PointCloudFilterUtils"))